keyvault/2016-10-01/swagger/keyvault.json

```diff
@@ -2547,7 +2547,7 @@
           "type": "array",
           "items": {
             "type": "string",
-            "description": "JSON web key operations. For more information, see JsonWebKeyOperation.",
+            "description": "JSON web key operations. The default is all operations. For more information, see JsonWebKeyOperation.",
```

Comment:
What do you mean by this: "The default is all operations. " ? I do not see any value in the enum that says "all operations" ?